config/profile.go

```diff
@@ -174,7 +174,7 @@ functionality - performance of content discovery and data
 fetching may be degraded.
 `,
 		Transform: func(c *Config) error {
-			c.Routing.Type = "dhtclient"
+			c.Routing.Type = NewOptionalString("dhtclient") // TODO: replace 'dhtclient' with 'autoclient' that prioritizes HTTP and uses dhtclient only after some delay as a fallback
```

Comment:
Can you make an issue for this and stick it on next iteration, so we can talk about its priority during planning?

Reply:
created https://github.com/ipfs/kubo/issues/9480